_scss/_content.scss

```diff
@@ -129,10 +129,9 @@ pre {
     font-size: 13px;
     line-height: 1.42857143;
     color: #333;
-    word-break: normal!important; 
-    word-wrap: break-word; 
+    word-break: normal!important;
+    word-wrap: break-word;
     background-color: #f5f5f5;
     border: 1px solid #ccc;
     border-radius: 4px;
-    overflow-x: scroll!important;
```

Comment:
What will happen now if we have long code lines? We don't want them to wrap because it can cause trouble in pasting.